From a37db0dd1aa80d83be5fa76e5b6b227b6bde8af9 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Tue, 16 Jun 2020 14:34:52 +0200
Subject: [PATCH 243/351] usb: typec: anx7688: Always enable VCONN

This is to avoid cabledet issues caused by leakage through VCONN
chargepump in PP 1.2a when the cable is unplugged.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 drivers/usb/typec/anx7688.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/typec/anx7688.c b/drivers/usb/typec/anx7688.c
index 3d2dd309d63a..0b6d3804fa96 100644
--- a/drivers/usb/typec/anx7688.c
+++ b/drivers/usb/typec/anx7688.c
@@ -149,7 +149,7 @@ static const char * const anx7688_supply_names[] = {
 };
 
 #define ANX7688_NUM_SUPPLIES ARRAY_SIZE(anx7688_supply_names)
-#define ANX7688_NUM_ALWAYS_ON_SUPPLIES (ANX7688_NUM_SUPPLIES - 2)
+#define ANX7688_NUM_ALWAYS_ON_SUPPLIES (ANX7688_NUM_SUPPLIES - 1)
 #define ANX7688_VCONN_INDEX (ANX7688_NUM_SUPPLIES - 2)
 #define ANX7688_VBUS_INDEX (ANX7688_NUM_SUPPLIES - 1)
 
-- 
2.34.0

